CHANGELOG.md

```diff
@@ -9,9 +9,7 @@
 
 ### Defects Corrected
 * Fix isHealthy field being included in the health dependency list JSON.
-
-### Defects Corrected
-* Fix isHealthy field being included in the health dependency list JSON.
+* Primary Health Check checking all dependencies.
```

Comment:
I would give more detail of the changes. Something like

```
* Health resource (/health) now excludes non-primary health dependencies. The non-primary dependencies are still included in the diagnostic resource (/health/diagnostic). The default for health dependencies is also changed to be primary.
```

Reply:
Changed it to your suggestion here: https://github.com/cerner/beadledom/pull/123/commits/7e25d081e7a14dc0701cb9573ab0645fbddd6414